.github/ISSUE_TEMPLATE/guidance-issue.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Guidance issue
+about: Create a report to help us improve
+title: ''
+labels: needs-triage, guidance
+assignees: ''
+
+---
+
+Please fill out the sections below to help us address your issue.
+
+**What issue did you see ?**
+
+**Steps to reproduce**
+If you have a runnable example, please include it as a snippet or link to a repository/gist for larger code examples.
+
+**Debug logs**
+Full stack trace by adding `boto3.set_stream_logger('')` to your code.
```

Comment:
Same comment here as in the `bug_report` template where we should be using the botocore version of `set_stream_logger`.